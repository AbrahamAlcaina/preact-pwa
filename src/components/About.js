import "@material/card/dist/mdc.card.css";
import "@material/button/dist/mdc.button.css";
import "./About.scss";

import { h } from "preact";

export default () => (
  <div class="about-component mdc-card">
    <section class="mdc-card__primary">
      <h1 class="mdc-card__title mdc-card__title--large">
        PWA
      </h1>
    </section>
    <section class="mdc-card__supporting-text">
      About this Application
    </section>
    <section class="mdc-card__actions">
      <a href="#/">
        <button class="mdc-button mdc-button--compact mdc-card__action">
          OK
        </button>
      </a>
    </section>
  </div>
);
