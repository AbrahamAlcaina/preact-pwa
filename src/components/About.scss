.about-component {
  &.mdc-card {
    width: 330px;
    height: 200px;
    text-align: center
  }

  .mdc-card__supporting-text {
    margin-top: 30px
  }

  .mdc-card__actions {
   margin: auto
  }
}