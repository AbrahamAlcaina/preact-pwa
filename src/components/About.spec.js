import { h } from "preact";
import render from "preact-render-to-string";
import About from "./About";

describe("About", () => {
  it("Renders correctly`", () => {
    expect(render(About)).toMatchSnapshot();
  });
});
