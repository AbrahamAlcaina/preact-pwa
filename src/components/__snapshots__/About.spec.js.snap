// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Renders correctly\` 1`] = `
"function () {return (
    (0, _preact.h)(&quot;div&quot;, { &quot;class&quot;: &quot;about-component mdc-card&quot; },
      (0, _preact.h)(&quot;section&quot;, { &quot;class&quot;: &quot;mdc-card__primary&quot; },
        (0, _preact.h)(&quot;h1&quot;, { &quot;class&quot;: &quot;mdc-card__title mdc-card__title--large&quot; }, &quot;PWA&quot;)),



      (0, _preact.h)(&quot;section&quot;, { &quot;class&quot;: &quot;mdc-card__supporting-text&quot; }, &quot;About this Application&quot;),


      (0, _preact.h)(&quot;section&quot;, { &quot;class&quot;: &quot;mdc-card__actions&quot; },
        (0, _preact.h)(&quot;a&quot;, { href: &quot;#/&quot; },
          (0, _preact.h)(&quot;button&quot;, { &quot;class&quot;: &quot;mdc-button mdc-button--compact mdc-card__action&quot; }, &quot;OK&quot;)))));}"
`;
